scripts/lib/CIME/XML/entry_id.py

```diff
@@ -254,7 +251,8 @@ def get_value(self, vid, attribute=None, resolved=True, subgroup=None):
         or from the values field if the attribute argument is provided
         and matches
         """
-        node = self.get_optional_node("entry", {"id":vid})
+        root = self.root if subgroup is None else self.get_optional_node("group", {"id":subgroup})
```

Comment:
Important change. Allows us to avoid "more than one match" errors when the user provided a subgroup